### PR TITLE
Updating `Upgrade running applications` to run with the latest SDK version

### PR DESCRIPTION
**Pull request by nemanja-da**
- Changed the initial `bash` script to set the latest SDK version for the `forum`project
- `create-daml-app` is now created with `daml new create-daml-app create-daml-app` commdn

### CODE REVIEW

File: upgrading/upgrade-ui/1_codegen.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ daml codegen js migration-0.1.0.dar -o daml.js
 Once finished, open `ui/package.json`{{open}} in the IDE and add the new dependencies for the
 `create-daml-app-0.1.1`, `forum-0.1.0` and `migration-0.1.0` packages
 
-<pre class="file" data-target="clipboard">
+```
```
Comment (ghost) [inline upgrading/upgrade-ui/1_codegen.md]: why was this changed?

File: upgrading/upgrade-ui/start_sandbox_background.sh [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+sdk_version=$(daml version | awk '/(default SDK version for new projects)/ {print $1}')
+echo "SDK version is " $sdk_version
```
Comment (ghost) [inline upgrading/upgrade-ui/start_sandbox_background.sh]: what's this for?

File: upgrading/upgrade-running-applications/init_background.sh [hunk 3]
```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
-
-tar xzf create-daml-app.tar.gz
+cd /tmp
```
Comment (ghost) [inline upgrading/upgrade-running-applications/init_background.sh:2]: did you figure out why you have to do this and if we could simply set the path instead?

**Review by ghost — APPROVED** [review]

File: upgrading/upgrade-ui/1_codegen.md (hunk 1 above)
Comment (nemanja-da) [reply upgrading/upgrade-ui/1_codegen.md]: Sigh, thought this would not get into the PR. I continued working on my master and didn't realise that upstream was set
